bin/omnia

```diff
@@ -11,17 +11,16 @@ cd "$(cd "${0%/*/*}" && pwd)/lib"
 . ./gasprice.sh
 . ./ethereum.sh
 . ./log.sh
-. ./source.sh
 . ./status.sh
 . ./util.sh
 . ./transport.sh
 . ./feed.sh
-. ./relayer.sh
+. ./relay.sh
 . ./omnia.sh
 
 initEnv
-if [[ "$OMNIA_MODE" == "RELAYER" || "$OMNIA_MODE" == "RELAY" ]]; then
```

Comment:
Can we keep "relayer" as an alias for "relay" to maintain backward compatibility